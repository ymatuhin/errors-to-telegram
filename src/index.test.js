const telebug = require('./index.js');

test('throw errors on init', () => {
  const chatId = '123';
  const botId = '123';

  expect(() => telebug()).toThrowErrorMatchingSnapshot();
  expect(() => telebug({ chatId })).toThrowErrorMatchingSnapshot();
  expect(() => telebug({ botId })).toThrowErrorMatchingSnapshot();

  expect(() => {
    telebug({ chatId, botId });
    telebug({ chatId, botId });
  }).toThrowErrorMatchingSnapshot();
});
