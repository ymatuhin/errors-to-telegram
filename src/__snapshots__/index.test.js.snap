// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`throw errors on init 1`] = `"❗️ Error: botId must be provided"`;

exports[`throw errors on init 2`] = `"❗️ Error: botId must be provided"`;

exports[`throw errors on init 3`] = `"❗️ Error: chatId must be provided"`;

exports[`throw errors on init 4`] = `"❗️ Error: telebug already inited"`;
